contrib/cirrus/packer/fah_setup.sh

```diff
@@ -8,9 +8,7 @@ set -e
 # Load in library (copied by packer, before this script was run)
 source /tmp/libpod/$SCRIPT_BASE/lib.sh
 
-req_env_var "
-SCRIPT_BASE $SCRIPT_BASE
-"
+req_env_var "$(basename $0)" "SCRIPT_BASE $SCRIPT_BASE"
```

Comment:
Need to remove the `basename`

Reply:
Actually there are still 21 instances of `req_env_var` that need to be fixed to the new usage. I think I'll hold off on further review until you resubmit.

Reply:
oh thanks for the reminder, actually all of FAH (atomic host) needs to go away...